pkg/cache/azure_cache.go

```diff
@@ -151,8 +151,7 @@ func (t *TimedCache) Get(key string, crt AzureCacheReadType) (interface{}, error
 
 	// set the data in cache and also set the last update time
 	// to now as the data was recently fetched
-	entry.Data = data
-	entry.CreatedOn = time.Now().UTC()
+	t.Update(key, data)
 
 	return entry.Data, nil
```

Comment:
`entry.Data` is `nil` or still refers to the expired value. Did you intend to `return data, nil` here?

Reply:
You are right, it should be `data, nil` here.